.github/workflows/release.yml

```diff
@@ -14,20 +14,21 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
```

Comment:
Hi @cpanato ! Curious why the hash instead of the the tag?

Reply:
it is a best practice to use the hash instead of the tag, the tag is mutable and can be changed, but the hash not :)
and we pin to a specific release. also dependabot can update that and also update the version comment :)


hope that clarifies

Reply:
Yes. Thanks!